src/App.jsx

```diff
@@ -19,9 +19,9 @@ export default function App() {
             <Route path="/login">
               <Login />
             </Route>
```

Comment:
Nice job making the route private utilizing built in react hooks to create context in your app to make is easy for every file to access your state.